flutter_dropzone_web/lib/assets/flutter_dropzone.js

```diff
@@ -40,26 +40,51 @@ class FlutterDropzone {
   drop_handler(event) {
     event.preventDefault();
 
-    var files = [];
-    var strings = [];
-    if (event.dataTransfer.items) {
+    // Handle array length and onMultipleDrop callback availability
+    if (event.dataTransfer.items.length < 1) {
+      // No dropped items in dataTransfer object
+      if (this.onError != null) this.onError("Empty data transfer");
```

Comment:
Why is empty payload considered an error?

Reply:
Because it is first step to make this package more strictly typed, remove dynamic from onDrop and onDropMultiple and assure it is non null.
Then we can also separate callbacks specifically for text/file drop.

Reply:
Can we just ignore it if it's empty?

Reply:
K, makes sense for me.